retailer-oas/retailer.yaml

```diff
@@ -459,63 +714,256 @@ components:
+
+    Schedule:
       type: object
       properties:
         id:
           type: string
-        # TODO: storeId? locationCode? pickingLocationId? placeId? coords?
-        # does the availability varies dependending on the kind of services? pick vs deliver vs click&collect vs ...
-        locationId: 
-          type: string
-        # date:
-        #   type: string
-        #   format: ISO 8601
-        timeSlot:
-          $ref: '#/components/schemas/TimeSlot'
-        capacity:
-          $ref: '#/components/schemas/Capacity'
-        available:
-          type: boolean
+        slots:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlotAvailability'
 
-    Schedule:
+    OrderArticle:
       type: object
       properties:
         id:
           type: string
-        slots: 
+          description: Unique identifier of the product
+          example: "12345"
+        sku:
+          type: string
+          description: Stock Keeping Unit of the product
+          example: "12345"
+        requestedQuantity:
+          type: number
+          description: The requested quantity of the product in the order
+          example: 10
+        unitOfMeasure:
+          type: string
+          description: The unit of measure of the product
+          example: "each"
+        name:
+          type: string
+          description: The name of the product
+          example: "Product A"
+        customerSpecialNote:
+          type: string
+          description: A description of any special request from the customer.
+          example: "I want it in small slices"
+        productImage:
+          type: string
+          format: uri
+          description: The URL of the product image
+          example: "https://example.com/productA.jpg"
+        longDescription:
+          type: string
+          description: The long description of the product
+          example: "This is a long description of Product A, including details about its features and specifications"
+        unitOfSale:
+          type: string
+          description: The unit of sale of the product
+          example: "case"
+        discount:
+          type: number
+          format: percent
+          description: The discount applied to the product
+          example: 0.1
+        buyingPrice:
+          type: number
+          format: currency
+          description: The buying price of the product
+          example: 10.99
+        unitPrice:
+          type: number
+          format: currency
+          description: The unit price of the product
+          example: 11.99
+        totalPrice:
+          type: number
+          format: currency
+          description: The total price of the product considering the quantity
+          example: 119.9
+        estimatedWeight:
+          type: number
+          format: weight
+          description: The estimated weight of the product
+          example: 12.5
+        replacementPreference:
```

Comment:
We should also include qualitative information -> frozen, fragile, hot and even things such as `hasAlcohol`.